core/jobs/src/main/java/com/cognifide/aet/job/common/utils/Sampler.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * AET
+ *
+ * Copyright (C) 2013 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.cognifide.aet.job.common.utils;
+
+import com.cognifide.aet.job.common.modifiers.resolution.ResolutionModifier;
+import java.util.LinkedList;
+import java.util.function.Supplier;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Sampler {
+
+  private static final Logger LOG = LoggerFactory.getLogger(ResolutionModifier.class);
```

Comment:
Did you intentionally pass `ResolutionModifier.class` instead of `Sampler.class` to `getLogger` method here?

Reply:
Ups, copy-paste fail. Will fix that.